routing/localchans/manager.go

```diff
@@ -85,6 +91,22 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 			return nil
 		}
 
+		// Mark this channel as found by removing it. chansToUpdate
+		// will be used to report invalid channels later on.
+		// Note: This must be executed before we look for inactive
+		// channels.
+		delete(chansToUpdate, info.ChannelPoint)
```

Comment:
The name on `chansToUpdate` seems a bit unintuitive it's actually the list of channels that we didn't update?

Reply:
The name is not intuitive, hence the comment above `delete`. I can create a separate list with a more appropriate name and copy all the necessary channel properties into it if you want? The rationale behind it was optimizing for memory usage.

Reply:
Hopefully I can find something more intuitive.